data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/dao/ScalarDBDao.java

```diff
@@ -0,0 +1,425 @@
+package com.scalar.db.dataloader.core.dataimport.dao;
+
+import com.scalar.db.api.*;
```

Comment:
Please don't use the wildcard `*`.

Reply:
I have updated to removed usage of wildcard.
Thank you